environment_gpu.yml

```diff
@@ -0,0 +1,16 @@
+name: null
+channels:
+  - pytorch
+dependencies:
+  - cudatoolkit=10.1
```

Comment:
I don't know what's up with that. 
Two things: 
1) If DLAMI already has cuda toolkit 10.2 and it works, then we should not pin to 10.1 but do a >=10.1. Then the doc is fine as is.
2) This should be raised as an engineering question/issue since I don't know the ramifications of the environment install/update not working as expected.

Reply:
We should definitely test this before committing. I'll try it both ways on the new DL AMI.